Octokit.Tests/Reactive/ObservableMilestonesClientTests.cs

```diff
@@ -76,15 +175,15 @@ public class TheGetForRepositoryMethod
                     }
                );
                 var gitHubClient = Substitute.For<IGitHubClient>();
-                gitHubClient.Connection.Get<List<Milestone>>(firstPageUrl, null, null)
+                gitHubClient.Connection.Get<List<Milestone>>(firstPageUrl, Args.EmptyDictionary, null)
                     .Returns(Task.Factory.StartNew<IApiResponse<List<Milestone>>>(() => firstPageResponse));
-                gitHubClient.Connection.Get<List<Milestone>>(secondPageUrl, null, null)
+                gitHubClient.Connection.Get<List<Milestone>>(secondPageUrl, Args.EmptyDictionary, null)
                     .Returns(Task.Factory.StartNew<IApiResponse<List<Milestone>>>(() => secondPageResponse));
-                gitHubClient.Connection.Get<List<Milestone>>(thirdPageUrl, null, null)
+                gitHubClient.Connection.Get<List<Milestone>>(thirdPageUrl, Args.EmptyDictionary, null)
                     .Returns(Task.Factory.StartNew<IApiResponse<List<Milestone>>>(() => lastPageResponse));
                 var client = new ObservableMilestonesClient(gitHubClient);
 
-                var results = await client.GetAllForRepository("fake", "repo").ToArray();
+                var results = client.GetAllForRepository("fake", "repo").ToArray().Wait();
```

Comment:
any reason to change this? Since all the other cases do the `await` rather than `.Wait()` ?